docs/website/blog/2023-05-30-odo-v3.11.0.md

```diff
@@ -0,0 +1,135 @@
+---
+title: odo v3.11.0 Released
+author: feloy
+author_url: https://github.com/feloy
+author_image_url: https://github.com/feloy.png
+image: https://raw.githubusercontent.com/redhat-developer/odo/main/docs/website/static/img/logo.png
+tags: ["release"]
+slug: odo-v3.11.0
+---
+
+odo `v3.11.0` is now out!
+
+<!--truncate-->
+
+To install `odo`, follow [the installation guide](../docs/overview/installation).
+
+## Notable Changes
+Check this Playlist for an overview of the most notable changes in this release:
+
+- https://www.youtube.com/playlist?list=PLGMB2PY4SNOqCTd7633dQ4DJv35TBaqHw
+- https://www.youtube.com/playlist?list=PLGMB2PY4SNOp_8zanRCxf2dsjMVkYyIJ6
+
+### Features
+
+#### [Devfile changes handled by odo dev on Podman](https://github.com/redhat-developer/odo/issues/6458)
+
+When a user is running odo against Podman, the changes to the Devfile is detected and applied, So the user can continuously iterate on the project without having to restart odo dev manually.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/T6m8bm9rYLw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+#### [Allow passing extra flags to Podman/Docker](https://github.com/redhat-developer/odo/issues/6421)
+
+`odo` is now able to pass extra flags to Podman or Docker backends, used for building images or starting a Dev session (Podman only).
+
+The main use case behind this is to allow users of `odo` to build images targeting a platform running a different architecture, e.g., users on arm64 architecture (like Mac M1) building images for a Kubernetes cluster that does not support arm64.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/MfGqsC2kkVM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+#### [Handling 'imageName' in Image Component as a selector](https://github.com/redhat-developer/odo/issues/6677)
+
+`odo` can now push images from Image Components to a given registry and use that for all matching Kubernetes/OpenShift components manifests, without having to hardcode that information into the Devfile, so that developers can keep their devfile portable.
```

Comment:
Suggestion to link to the actual doc page to learn more about this.

```suggestion
`odo` can now push images from Image Components to a given registry and use that for all matching Kubernetes/OpenShift components manifests, without having to hardcode that information into the Devfile, so that developers can keep their devfile portable.

See [How `odo` handles image names](../docs/development/devfile#how-odo-handles-image-names) for more details.
```